### ISSUE TITLE
Remove lib/example.rb and spec/example_spec.rb

### ISSUE THREAD

**Issue by raq929** [ga-wdi-boston/ruby-objects-composition#20]
I believe our policy is to remove these files once other specs have been added.

**Comment by jrhorn424** [comment]
> I believe our policy is to remove these files once other specs have been added.

Yes. I'd say this captures the spirit of the policy: "Either example files, or a .gitkeep file, should be included to maintain a common directory structure, but should be removed when no long necessary."
